findingcheapestgpus.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Finding the Cheapest GPUs'
+---
+Shadeform makes it easy to find and deploy VM and containers to the cheapest GPU instances across many different clouds. Let's show how we can do this with the UI and the API, specifically for an instance with a single 80 GB A100 GPU.
+
+
+## Using the Shadeform UI
+
+### 1. Searching for GPU's
+Once you've created an account and added funds to your [Wallet](https://platform.shadeform.ai/settings/billing) then you can see available instances on the homepage. 
+
+![GPU Filter](images/1x80GBA100.png)
+
+### 2. Launching a GPU
+We can select for a single 80 GB A100 as shown. Listed are the instances that are currently available. We can select to launch one.
+![Launching](images/launchgpu.png)
+### 3. Waiting for startup
+When we go into the Running instances [tab](https://platform.shadeform.ai/running), we can see that this VM is pending while it spins up.
+![Pending](images/pending.png)
+### 4. Scheduling an instance
+If we want an even cheaper instance but are willing to wait, we can also Schedule and instance instead. 
```

Comment:
Can we mention availability here? For ex. "If we want an even cheaper instance that's not available and we are willing to wait, we can Schedule to launch that instance if it becomes available during the specified time period". Feel free to reword

Reply:
I assume this was addressed, by the section below (#4)